registration/registration-services/src/main/java/io/mosip/registration/service/external/impl/PreRegZipHandlingServiceImpl.java

```diff
@@ -107,7 +107,7 @@ public RegistrationDTO extractPreRegZipFile(byte[] preRegZipFile) throws RegBase
 
 			try (ZipInputStream zipInputStream = new ZipInputStream(new ByteArrayInputStream(preRegZipFile))) {
 				ZipEntry zipEntry;
-				while ((zipEntry = zipInputStream.getNextEntry()) != null) {
+				while ((zipEntry = zipInputStream.getNextEntry()) != null) {    //NOSONAR Adding Next ZipEntry here.
```

Comment:
what was the issue reported related to ZipEntry?

Reply:
Security hotspot with low priority is identified as below:
Make sure that expanding this archive file is safe here.
Expanding archive files without controlling resource consumption is security-sensitive[java:S5042](https://sonarcloud.io/organizations/mosip/rules?open=java%3AS5042&rule_key=java%3AS5042)